src/portal/src/lib/components/config/gc/gc.component.scss

```diff
@@ -1,15 +1,29 @@
 .cron-selection {
-    padding-left: 0.25rem;
+    margin-top: 1rem;
```

Comment:
![image](https://user-images.githubusercontent.com/29826789/84230521-b0b14e80-ab1e-11ea-98df-9e10c615b814.png)
#12147 has not been changed

Reply:
Fixed